ms2/src/org/labkey/ms2/MS2Module.java

```diff
@@ -318,9 +318,9 @@ public Collection<String> getSummary(Container c)
 
     @Override
     @NotNull
-    public Set<String> getSchemaNames()
+    public List<String> getSchemaNames()
     {
-        return PageFlowUtil.set(MS2Schema.SCHEMA_NAME, ProteinManager.getSchemaName());
+        return List.of(ProteinManager.getSchemaName(), MS2Schema.SCHEMA_NAME);

```

Comment:
I assume this ensures that `prot` is created first so it can be referenced by FKs from `ms2`?